tools/lib/gzipTileset.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+var Cesium = require('cesium');
+var fsExtra = require('fs-extra');
+var path = require('path');
+var Promise = require('bluebird');
+var zlib = require('zlib');
+
+var fsExtraCopy = Promise.promisify(fsExtra.copy);
+var fsExtraEnsureDir = Promise.promisify(fsExtra.ensureDir);
+var fsExtraReadFile = Promise.promisify(fsExtra.readFile);
+
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+var DeveloperError = Cesium.DeveloperError;
+
+module.exports = gzipTileset;
+
+/**
+ * gzips or gunzips the input tileset.
+ *
+ * @param {String} inputDirectory Path to the input directory.
```

Comment:
Consider putting this all in the `options` object (like Cesium) or maybe strictly follow a pattern where required parameters are not in `options` and required ones are.  Ask I mentioned above, I think `outputDirectory` should be optional.